docs/source/index.mdx

```diff
@@ -1,19 +1,13 @@
-# `bitsandbytes`
+# bitsandbytes
 
-The `bitsandbytes` library is a lightweight Python wrapper around CUDA custom functions, in particular 8-bit optimizers, matrix multiplication (LLM.int8()), and 8 + 4-bit quantization functions.
+bitsandbytes makes large language model inference and training more accessible by dramatically reducing memory consumption with 8-bit optimizers and k-bit quantization.
 
-The library includes quantization primitives for 8-bit & 4-bit operations, through `bitsandbytes.nn.Linear8bitLt` and `bitsandbytes.nn.Linear4bit` and 8bit optimizers through `bitsandbytes.optim` module.
-
-There are ongoing efforts to support further hardware backends, i.e. Intel CPU + GPU, AMD GPU, Apple Silicon. Windows support is on its way as well.
-
-## API documentation
-
-- [Quantization](quantization)
-- [Integrations](integrations)
-- [Optimizers](optimizers)
+* 8-bit optimizers uses block-wise quantization to maintain 32-bit performance at a small fraction of the memory cost.
+* LLM.Int() or 8-bit quantization enables large language model inference with only half the required memory and without any performance degradation. This method is based on vector-wise quantization to quantize most features to 8-bits and separately treating outliers with 16-bit matrix multiplication.
+* QLoRA or 4-bit quantization enables large language model training with several memory-saving techniques that don't compromise performance. This method quantizes a model to 4-bits and inserts a small set of trainable low-rank adaptation (LoRA) weights to allow training.
 
 # License
 
-The majority of bitsandbytes is licensed under MIT, however portions of the project are available under separate license terms, as the parts adapted from Pytorch are licensed under the BSD license.
+The majority of bitsandbytes is MIT licensed. However, portions of the project are available under separate license terms such as the parts adapted from PyTorch which are licensed under the BSD license.
```

Comment:
I'm a little curious - do we know specifically which parts are adapted from PyTorch?

Reply:
Sorry, I'm not too sure which parts are specifically adapted from PyTorch. Still finding my way around the library 🙂 

I think @Titus-von-Koeller or @younesbelkada can help answer?

Reply:
Hey! Yeah, I already pinged Tim about ti. Also about the licensing stuff in general. From what I understand it's only 8 lines of code from PyTorch but I don't know which, yet. I'll get back to you on this.